.github/workflows/release-docker-image-server-only.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release UID2 Examples Image for Server Only
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} Server Only Docker Image by @${{ github.actor }}  
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: 'The type of release'
+        options:
+        - Major
+        - Minor
+        - Patch
+        - Snapshot
+  pull_request:
+
+jobs:
+  incrementVersionNumber:
+    uses: IABTechLab/uid2-shared-actions/.github/workflows/shared-increase-version-number.yaml@kcc-UID2-2674-implement-shared-publish-to-docker-versioned
+    with:
+      release_type: ${{ inputs.release_type }}
+      working_dir: publisher/server_only
+    secrets: inherit
+
+  publishForServerOnly:
```

Comment:
This could just be a step in the above job - no reason to have 2 different jobs. Makes running slower